feature-group-definitions/array-by-copy.yml

```diff
@@ -0,0 +1,30 @@
+name: Array by copy
+description: Array and typed array by copy methods toReserved(), toSorted(), toSpliced(), with()
```

Comment:
```suggestion
description: Array and typed array by copy methods toReserved(), toSorted(), toSpliced(), and with()
```

Or, alternatively:

```suggestion
description: Array and typed array by copy methods (toReserved(), toSorted(), toSpliced(), with())
```

Reply:
FWIW, the first suggestion is closer to what I've done elsewhere.